06-model-infer/01-lesson/06-01-lesson.Rmd

```diff
@@ -161,9 +161,9 @@ Indeed, when we take repeated samples of size 20 (here we took 50 different samp
 
 
 ```{r echo = FALSE}
-starbucks_many_lm <- starbucks_many %>% 
-  group_by(replicate) %>% 
-  do(tidy(lm(Calories ~ Fat, data=.))) %>%
+starbucks_many_lm <- starbucks_many |> 
+  group_by(replicate) |> 
+  do(tidy(lm(Calories ~ Fat, data=.))) |>
```

Comment:
Does this work as expected?

Reply:
It does. I ran all code chunks and compared to the online tutorial and output is the same.